### DISCUSSION TITLE
Where do i put the config to set the Char

### DISCUSSION THREAD

**Discussion by Fish1** [lukas-reineke/indent-blankline.nvim#714] [General]
In my old setup for v2 I just had lazy vim and set opt = { char = "|" }

Where exactly do I need to set this new "ibl.config.indent.char" variable, so I can get my setup working again?

I tried this... probably completely wrong... vim noob here.

```lua
local ibl = require("ibl")
ibl.setup()
ibl.config.indent.char = "|"
```

**Comment by lukas-reineke** [comment]
It should be
```
ibl.setup {
  indent = {
    char = "|"
  }
}
```

**Reply by Fish1** [reply]
Thank you!
